aw_transform/classify.py

```diff
@@ -45,6 +50,7 @@ def categorize(events: List[Event], classes: List[Tuple[Category, Rule]]):
 
 
 def _categorize_one(e: Event, classes: List[Tuple[Category, Rule]]) -> Event:
+    # TODO can we add a color here too? why is color rendered on the frontend?
```

Comment:
We're keeping the coloring in the frontend to try and keep the server as lean as possible.

```suggestion
```

Reply:
Removed.